ci-operator/config/openshift/cluster-authentication-operator/openshift-cluster-authentication-operator-master.yaml

```diff
@@ -48,5 +48,6 @@ tests:
 - artifact_dir: /tmp/artifacts
   as: e2e-aws-console-login
   commands: ./test-prow-e2e.sh login
-  openshift_installer_console:
+  openshift_installer_custom_test_image:
     cluster_profile: aws
+    test_imagestream_tag: stable:console-tests
```

Comment:
I'd prefer `from` as the key name to parallel the container test type and would not expect people to need to specify `stable` -- that's an internal detail

Reply:
as you wish i guess, but i think my name choice offers more context to an inexperienced observer/reader of the file.

Reply:
updated https://github.com/openshift/ci-tools/pull/24 w/ this change, and updated this PR to reflect it.